3-projects/folders.tf

```diff
@@ -14,7 +14,19 @@
  * limitations under the License.
  */
 
-resource "google_folder" "app" {
-  parent       = "organizations/${var.organization_id}"
-  display_name = "app"
+/******************************************
+  Business Unit Folders
+ *****************************************/
+resource "google_folder" "business_unit" {
+  display_name = "example-business-unit"
+  parent       = "organizations/${var.org_id}"
+}
+
+/******************************************
+  Team Folders
+ *****************************************/
+module "standard_project_folders" {
```

Comment:
for consistency, it probably makes sense to call this something like `example_team_folders`